src/index.js

```diff
@@ -81,13 +81,18 @@ function makeKCDWorkshopApp({
     }
   }
 
+  function escapeForClassList(name) {
+    // classList methods don't allow space or `/` characters
+    return encodeURIComponent(name.replace(/\//g, '_'))
+  }
+
   function handleLocationChange(location = history.location) {
     const {pathname} = location
     // add location pathname to classList of the body
     document.body.classList.remove(
-      previousLocation.pathname.replace(/\//g, '_'),
+      escapeForClassList(previousLocation.pathname),
     )
-    document.body.classList.add(pathname.replace(/\//g, '_'))
+    document.body.classList.add(escapeForClassList(pathname))
```

Comment:
Do you remember the purpose of changing `classList` of the body inside iframe?

I preserved the `.replace(/\//g, '_')` part in the function for backwards compatibility, but if there is no CSS to target some specific className and it's used for something else, then it might not be needed.